### PR TITLE
add support for root certificates

### PR DESCRIPTION
**Pull request by PratikMahajan**
uses all the certificates in the directory to make calls. 
this will ensure that, if certificates exist, we'll use them.

### CODE REVIEW

File: cincinnati/src/plugins/internal/graph_builder/commons.rs [hunk 1]
```diff
@@ -1,3 +1,60 @@
...
+    let mut certs: Vec<Certificate> = Vec::new();
+    let mut dirs: Vec<(PathBuf, u8)> = vec![(dir.to_path_buf(), 0)];
+    while !dirs.is_empty() {
+        let (dir, level) = dirs.pop().unwrap();
+        if let Ok(cert_paths) = std::fs::read_dir(dir) {
+            for path in cert_paths {
+                if let Ok(path) = path {
+                    if path.metadata().unwrap().is_file() {
+                        match path.path().extension() {
+                            Some(extension) => {
```
Comment (petr-muller) [inline cincinnati/src/plugins/internal/graph_builder/commons.rs:21]: I'm not a fan of conditionals nested this deep but I don't see an obvious way to improve it significantly to be worth the hassle so let's keep it I guess, given it implements a rather straightforward thing :shrug:

**Review by petr-muller — APPROVED** [review]
LGTM, holding to allow more reviews if needed

/hold

**Review by LalatenduMohanty — APPROVED** [review]
/lgtm

**Pull request merged**